src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -228,7 +266,8 @@ private void LoadData()
                     valueList.Add(dataValue);
                 }
 
-                string displayString = GridViewHelpers.GetPaddedString(valueList, _listViewOffset, _listViewColumnWidths);
+                string displayString = GridViewHelpers.GetPaddedString(
+                        valueList, _gridViewDetails.ListViewOffset, _gridViewDetails.ListViewColumnWidths);
```

Comment:
NIT: see no value to have the params start on a new line here, then it's indented too much

```suggestion
                string displayString = GridViewHelpers.GetPaddedString(valueList, _gridViewDetails.ListViewOffset, _gridViewDetails.ListViewColumnWidths);
```

Reply:
CodeFactor in PowerShell repo suggests to put all parameters on one line or every parameter on separate line.